### ISSUE TITLE
Metrics exporter

### ISSUE THREAD

**Issue by maximeancellin** [YOURLS/charts#276]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Self troubleshooting

- [X] I've read the [documentation](https://yourls.org/) and made sure to follow it.
- [X] I've read the [troubleshooting first steps](https://docs.yourls.org/guide/troubleshooting/first-steps.html) and [frequent issues](https://docs.yourls.org/guide/troubleshooting/frequent-issues.html) guides.

### Version

1.9.2

### Description

Hi, 

I would like to report an error on metrics exporter configuration embedded with the chart.

When we use the Helm chart and set metrics in `metrics.enabled` on `true`.
The exporter can't fetch Apache2 status page and returned following error logs:
```
"Error scraping apache:" err="error scraping apache: Get \"http://status.localhost:8080/server-status/?auto\": dial tcp 127.0.0.1:8080: connect: connection refused"
```

I have tried to curl the host inside the YOURLS container and I have the same result.
In Apache configuration I can't find a reference about port `8080`.

Thanks in advance for your support and I stay available if you need any more precision.

### Expectation

We would like to have Apache metrics return without error of the exporter.

Currently we have `apache_up` metrics in Prometheus with `0` value.

But YOURLS working fine.

Basically, the result should be `1`.

### Reproduction steps

I use the following values with the chart:

```yaml
nameOverride: "yourls"
fullnameOverride: "yourls"

image:
  tag: 1.9.2

yourls:
  domain: url.com
  scheme: https
  existingSecret: "yourls-admin"
  tablePrefix: yourls_
  skipInstall: false

extraEnvVarsSecret: yourls

replicaCount: 2
resources:
  limits:
    memory: 512Mi
  requests:
    memory: 512Mi

service:
  type: ClusterIP

ingress:
  enabled: true
  ingressClassName: nginx-public
  hostname: url.com
  path: /
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt-prod
    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
    nginx.ingress.kubernetes.io/ssl-redirect: "true"
  tls: true
  extraTls: []

persistence:
  enabled: false

metrics:
  enabled: true
  serviceMonitor:
    enabled: true

mariadb:
  enabled: false

podAntiAffinityPreset: hard

externalDatabase:
  host: "34.140.67.101"
  port: 3306
  user: urlshort
  database: yourls
  existingSecret: "yourls-db"
```

I'm working with GKE on version `1.24`.

**Comment by LeoColomb** [comment]
Thanks for opening this issue @maximeancellin, and thank you for your patience.
It might take some more time to be able to investigate, but if I recall correctly, this is simply not configured yet. I'm not even sure the `mod_status` module is loaded.

The main task is to load a specific vhost config that enables localhost metrics endpoint, something similar to this one: https://github.com/bitnami/containers/blob/2de62bea5b8ac9c5b2b35adf3afdf731d1a752b6/bitnami/wordpress/6/debian-11/rootfs/opt/bitnami/apache/conf/vhosts/00_status-vhost.conf#L4

Pull request welcomed of course 😊

**Comment by LeoColomb** [comment]
@maximeancellin Happy to have this resolved _just_ before a year span ✨
It should be fixed with version 6.1.0.

**Comment by halkeye** [comment]
so, apache isn't listening on 8080, its listening on 80

should https://github.com/YOURLS/charts/blob/2f44fa9966725dc51612c0d040422b6daa7aa2d5/charts/yourls/templates/deployment.yaml#L234 not be 80?
